### ISSUE TITLE
Plans for website

### ISSUE THREAD

**Issue by marcustisater** [postcss/postcss.org#246]
@ai the last real update for this website was over a year ago, and I haven't had the time to keep this website up-to-date with the Phenomic CMS or with new features. 

Personally, I think if anyone would like to rebuild this website in either Gatsby or update Phenomic, it's more then welcome. I think that would make the website more maintainable. 

What do you think? 😄

**Comment by ai** [comment]
I like the idea. I think we could keep current design and just move to pure HTML. Anyway, it is a static website, we do not need even Gatsby. Just webpack for performance optimization.

**Comment by okonet** [comment]
May I propose @bebraw [Antwar](https://antwar.js.org) which is used for [React Finland](https://react-finland.fi) and [ReasonConf](https://reason-conf.com) websites? It has all the required features afaik and allows dynamic (and highly optimized) components written in React which we'd need for the contributor's list.

**Comment by bebraw** [comment]
Let me know if you need anything specific. Now it's coupled with React/react-router/webpack but this can be changed. The output is entirely static by default by design and there's a solution for interactivity (think progressive enhancement) should you happen to need it.

The gotcha is that you won't get Gatsby style routing. Also Gatsby ecosystem is far stronger and you are more likely to find a contractor to do the work unless you want to tackle this yourself.

I can support to an extent and help with the overall architecture depending on your exact needs. I am mostly curious things like authoring flow and information architecture, and the way you want to layout the site as those will have implications on the site design.

**Comment by marcustisater** [comment]
Yes, we definitely need to update the website to maintain it better, it's difficult for developers to set it up today on their local machine, outdated node etc. We have talked before about adding static information pages and other stuff for "milestone 2". Let's add issues to it https://github.com/postcss/postcss.org/milestone/5

I would love to take on the project with some help. @okonet how is your schedule looking? I might need help with design and React guidance. 

Thank you @bebraw 

@ai what are you thoughts regarding this?
